### ISSUE TITLE
Questions about fine-tuning with bridgeData v2 and evaluation results

### ISSUE THREAD

**Issue by shiyouheng** [OpenDriveLab/UniVLA#55]
Hello! First of all, thank you very much for developing and open-sourcing this project. It has provided valuable research resources to the community and saved us a lot of development time. During usage, I encountered a few questions I'd like to ask you:

1. I noticed that the `finetune_bridge.py` file seems to be missing in the project. Could you please share if there are any special designs or recommended configurations for experimental hyperparameters (such as learning rate, batch size, etc.) when fine-tuning with bridgeData v2?

2. When evaluating on SimplerEnv, I found that History latent actions are not used, and only task descriptions are used as language instructions. Does this mean that we should follow the same setup during the fine-tuning phase with bridgeData v2, i.e., not using History latent actions?

Looking forward to your reply, and thank you again for your open-source contribution!
